### PR TITLE
Apply new scalafmt rules

### LINKED ISSUE #34: Formatting discussion

### PR DESCRIPTION
**Pull request by legopiraat**
name: Apply new scalafmt rules
about: Formatting & common guides for the repository

#### What it does

Formats all the files based on a new set of rules found in: `.scalafmt.conf`

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -1,2 +1,12 @@
-maxColumn = 100
-project.git = true
\ No newline at end of file
+align = more
```
Comment (hugo-vrijswijk) [inline .scalafmt.conf]: Not sure I am a fan of this (mostly having assignment `=`'s aligned). It's fairly unstandard I think. I would prefer `align = some`, [which is the default](https://scalameta.org/scalafmt/docs/configuration.html#align-some) to align case arrows, but not much more

File: sbt/src/main/scala/stryker4s/sbt/SbtMutantRunner.scala [hunk 2]
```diff
@@ -2,8 +2,6 @@ package stryker4s.sbt
 import java.nio.file.Path
 
 import better.files.File
-import sbt.Keys._
```
[the remark this answers is not in the document]
Comment (legopiraat) [reply sbt/src/main/scala/stryker4s/sbt/SbtMutantRunner.scala]: Ow wauw that is probably also the reason the build failed i didnt notice

File: .scalafmt.conf (hunk 1 above)
Comment (legopiraat) [reply .scalafmt.conf]: Hmm it seems i read it wrong somehow i thought none was the default, i was already doubting between some and more so ill change it to some :)

**Comment by legopiraat** [comment]
@hugo-vrijswijk you think we should maybe just use the RC version for now? and revert the rules we want you didnt agree on?

File: .scalafmt.conf [hunk 3]
```diff
@@ -1,2 +1,12 @@
...
+align.openParenDefnSite = true
+
+lineEndings = unix
+
+maxColumn = 120
```
[the remark this answers is not in the document]
Comment (legopiraat) [reply .scalafmt.conf]: I think 120 is big enough :)

**Comment by hugo-vrijswijk** [comment]
Yeah I think using RC is fine. I'll have a look later (hopefully) today on what specific rules are changed

**Comment by legopiraat** [comment]
@hugo-vrijswijk this good enough for now? We can upgrade to 2.0.0 when it's out as a sbt plugin

**Pull request merged**